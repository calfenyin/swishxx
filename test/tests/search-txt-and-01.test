search | | | abominable year | 0
