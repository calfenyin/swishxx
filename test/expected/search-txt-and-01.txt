# results: 1
100 ./Time_Machine,_The.txt 182203 Time_Machine,_The.txt
